src/features/admin/external-evaluators/external-evaluators-table.tsx

```diff
@@ -73,7 +73,8 @@ export const ExternalEvaluatorsTable = () => {
       <Dialog open={showDialog}>
         <Dialog.Title>Delete External Evaluator</Dialog.Title>
         <Dialog.Description>
-          Are you sure you want to delete this evaluator? <br /> This action cannot be reverted.
+          Are you sure you want to delete this evaluator? <br />
+          This will delete all evaluations associated with this evaluator.
```

Comment:
I think we should add `"and cannot be reverted."` according to the attached picture in the said ticket?

![image](https://github.com/nerubia/kh360-react/assets/91468846/5b6130b8-8a83-47c9-9990-d7c838d85375)